pkg/bbgo/indicator_set.go

```diff
@@ -73,34 +76,34 @@ func (i *IndicatorSet) VOLUME(interval types.Interval) *indicatorv2.PriceStream
 	return indicatorv2.Volumes(i.KLines(interval))
 }
 
-func (i *IndicatorSet) RSI(iw types.IntervalWindow) *indicatorv2.RSIStream {
-	return indicatorv2.RSI2(i.CLOSE(iw.Interval), iw.Window)
+func (i *IndicatorSet) RSI(iw types.IntervalWindow) *momentum.RSIStream {
+	return momentum.RSI2(i.CLOSE(iw.Interval), iw.Window)
```

Comment:
I think it's okay to put all indicators in the same package (v2) since it sometimes might use each other as the source

Reply:
Once I am done that directory will have hundreds of files and I liked the way it gives a little more descriptive context to what the indicator type is eg momentum.RSI.
Besides I found that all indicator derivatives so far reference the base pkg or mostly trend (ma) indicator types with exception of subtract but I gonna fix the test to be more generic.
It's your call I can certainly squash them together in the indicatorv2 package just think it's tedious to work with a gazillion files in the ide. What do you think given those arguments?

Reply:
Yeah I agree with you from the development perspective, when developing these indicators, it's good to have these indicators to be categorized.

from the strategy user's perspective, they will have to look up the indicators from different packages in the source code. Originally, this could be done by simply importing `indicatorv2`, and the IDE shall show all the v2 indicators that can be used in one's strategy.

that's also the reason that we put the major API component under the pkg/bbgo package, which simplifies the imports in the strategy.

it would be great if we could categorized these indicators but also keep the simple package importing. ;-)

Reply:
done